service/controller/azure_config.go

```diff
@@ -0,0 +1,656 @@
+package controller
```

Comment:
This isn't new. This is the controller that we already had, the one that watches `AzureConfig` CRs, but renamed from `cluster.go` to `azure_config.go`. There are no changes in this file besides naming.